examples/bls/Containerfile.ubuntu

```diff
@@ -0,0 +1,82 @@
+# This is an experimental example
+#
+## Run built image directly
+##  cfsctl oci pull docker://docker.io/shawn111/cfs-images:ubuntu
+##  cfsctl oci prepare-boot [image id]
+
+
+## composefs part
+##
+##  build composefs related binaries, like mkcomposefs
```

Comment:
This part can definitely go now.  We have the erofs writer support integrated directly into cfsctl so we no longer need mkcomposefs in the kernel.

Reply:
@allisonkarlitskaya I rebased from main branch and kernel 6.14 is already in repo. It is much simple now.